### ISSUE TITLE
Placeholder shouldn't be considered the aria-label

### ISSUE THREAD

**Issue by ssorallen** [Polymer/core-input#56]
The HTML5 spec mentions the ["placeholder" attribute](http://www.w3.org/html/wg/drafts/html/CR/forms.html#the-placeholder-attribute) should not be used as a replacement for a label. However, core-input always adds an "aria-label" attribute set to the element's "placeholder" attribute.

This seems like it should be left to the developer. Use a real `<label for="">` and add an `aria-labelledby` attribute to the `<input is="core-input">` manually. If there is no real `<label>` for the input, manually add an `aria-label` attribute to the element that is labeling the input.
